bin/getjsonconfig.py

```diff
@@ -0,0 +1 @@
+get-json-config.py
```

Comment:
What is this single line file doing?

Reply:
getjsonconfig.py  is a symlink to get-json-config.py, without the dashes. This is because you can't import modules named with dashes with import <module_name>. The solution is to rename the file and meet the PEP8 requirements. However, we are calling this somewhere else in the rickshaw-run script, so we need to change there first and then we rename the file here.